assets/airports/ksfo.json

```diff
@@ -127,27 +127,27 @@
     {
       "name":        ["28R", "10L"],
       "name_offset": [[0, 0.3], [0, 0.3]],
-      "end":    [ ["N37.6135324", "W122.3571410"], ["N37.6287386", "W122.3933911"]],
+      "end":    [ ["N37.6135324", "W122.3571410", "13.0ft"], ["N37.6287386", "W122.3933911", "5.5ft"]],
```

Comment:
@erikquinn, I pulled them from https://www.airnav.com/airport/KSFO which indicates the field elevation is surveyed and I assumed the runway elevations are too.